### ISSUE TITLE
Quick start tutorial not working anymore after opendj was upgraded

### ISSUE THREAD

**Issue by soisik** [lsc-project/lsc#71]
Steps to reproduce:
- Follow instructions in quick-start tutorial https://lsc-project.org/documentation/latest/sample (I used jdk 8 and LSC 2.1.4)
- When starting openDj server, you get an error : 

```
2116 [main] ERROR org.lsc.opendj.LdapServer - org.opends.server.util.LDIFException: Entry dc=lsc-project,dc=org read from LDIF starting at line 1 is not valid because it violates the server's schema configuration:  Entry dc=lsc-project,dc=org violates the Directory Server schema configuration because it does not include a structural objectclass.  All entries must contain a structural objectclass
```
- After adding objectclass: organization in first entry of sample/hsqldb/etc/test.ldif, the quickstart works OK.

PR will follow.

**Comment by soisik** [comment]
Issue was corrected already in 2.1.4 branch in july by a contributor, but it has not been released yet. I've made a format fix (grouping attributes objectClass together)

**Comment by coudot** [comment]
Indeed we need to release 2.1.5 soon.
